content/en/security/cloud_security_management/_index.md

```diff
@@ -33,33 +33,59 @@ further_reading:
     text: "Customize rules for detecting cloud misconfigurations with Datadog Cloud Security Management"
 ---
 
-## Overview
+Datadog Cloud Security Management (CSM) delivers real-time threat detection and continuous configuration audits across your entire cloud infrastructure, all in a unified view for seamless collaboration and faster remediation. Powered by observability data, security teams can determine the impact of a threat by tracing the full attack flow and identify the resource owner where a vulnerability was triggered.
 
-Datadog Cloud Security Management delivers real-time threat detection and continuous configuration audits across your entire cloud infrastructure - in a unified view for seamless collaboration and faster remediation. Powered by observability data, security teams can determine the impact of a threat quickly by tracing the full attack flow and identify the resource owner where a vulnerability was triggered. Engineers are able to actively monitor their security risks by incorporating security metrics into their existing workflow.
+CSM leverages the Datadog Agent and platform-wide cloud integrations and includes:
 
-Cloud Security Management includes [Cloud Security Posture Management (CSPM)](#cloud-security-posture-management), and [Cloud Workload Security (CWS)](#cloud-workload-security).
+- [**Threats**][1]: Monitors file, network, and process activity across your environment to detect real-time threats to your infrastructure.
+- [**Misconfigurations**][2]: Tracks the security hygiene and compliance posture of your production environment, automates audit evidence collection, and enables you to remediate misconfigurations that leave your organization vulnerable to attacks.
```

Comment:
The left nav shows these in the other order: Misconfig and then Threats. Would it make sense to use the same order in both places?

Reply:
Good point. I'll update the order in the left nav (since the above is intended to match the order in the UI).